share/jupyter/nbconvert/templates/latex/style_jupyter.tex.j2

```diff
@@ -95,6 +95,7 @@
     \definecolor{outcolor}{HTML}{D84315}
     \definecolor{cellborder}{HTML}{CFCFCF}
     \definecolor{cellbackground}{HTML}{F7F7F7}
+    \definecolor{outerrorbackground}{HTML}{FFDFDF}
```

Comment:
This line should go in the document contents. Otherwise this will break other cell styles.

Reply:
Do you think you could help fix that up? Or should we make a new issue and tag it for the next alpha release?